cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslReferenceContent.groovy

```diff
@@ -0,0 +1,151 @@
+package org.jenkinsci.plugins.workflow.cps.Snippetizer
+
+import org.jenkinsci.plugins.workflow.cps.GlobalVariable
+import org.jenkinsci.plugins.workflow.cps.Snippetizer
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor
+import org.jenkinsci.plugins.workflow.structs.DescribableHelper
+
+import javax.servlet.RequestDispatcher
+
+Snippetizer snippetizer = my;
+
+def l = namespace(lib.LayoutTagLib)
+def st = namespace("jelly:stapler")
+
+
+h1(_("Steps"))
+for (StepDescriptor d : snippetizer.getStepDescriptors(false)) {
```

Comment:
I just really wanted to avoid having any nontrivial logic in Groovy views. There is limited IDE support, no build-time type checking, etc. That is why I had the original HTML generation in Java code. The one problem that approach had was that I could not figure out how to call from Java code back into Stapler for rendering help views of global variables (the `st.include(page: "help", it: v)` here).